### PR TITLE
Add support for custom URLs for each side

### PR DESCRIPTION
**Pull request by ip**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,14 +31,21 @@ AFRAME.registerComponent('cubemap', {
 
     // Path to the folder containing the 6 cubemap images
     var srcPath = data.folder;
+    if (srcPath === '')
```
Comment (bryik) [inline index.js:34]: Style-wise I prefer to always use brackets in `if` statements (even if they're not strictly necessary).

File: index.js [hunk 2]
```diff
@@ -28,14 +31,21 @@ AFRAME.registerComponent('cubemap', {
...
-      'posy.jpg', 'negy.jpg',
-      'posz.jpg', 'negz.jpg'
-    ];
+    var urls = this.data.urls;
+    if (urls.length === 0)
+      urls = [
+        'posx.jpg', 'negx.jpg',
+        'posy.jpg', 'negy.jpg',
+        'posz.jpg', 'negz.jpg'
+      ];
```
Comment (bryik) [inline index.js:45]: brackets here too, please

**Review by bryik — CHANGES_REQUESTED** [review]
This looks like a nice feature.

Would you mind adding an example as well?

File: index.js [hunk 3]
```diff
@@ -28,14 +31,21 @@ AFRAME.registerComponent('cubemap', {
...
+      srcPath = undefined;
 
     // Cubemap image files must follow this naming scheme
     // from: http://threejs.org/docs/index.html#Reference/Textures/CubeTexture
-    var urls = [
-      'posx.jpg', 'negx.jpg',
-      'posy.jpg', 'negy.jpg',
-      'posz.jpg', 'negz.jpg'
-    ];
+    var urls = this.data.urls;
```
Comment (bryik) [inline index.js:39]: You can just do `data.urls;`

I mentioned this in my review, but it didn't get added. Strange.

**Comment by ip** [comment]
Okay, I'll fix these a bit later and add an example. Also, I forgot to mention this in README.

**Comment by bryik** [comment]
Closing as this has been open for a while.

**Pull request closed without merging**